### DISCUSSION TITLE
About the plugins on Windows?

### DISCUSSION THREAD

**Discussion by Artist-on-line** [audacious-media-player/audacious#1498] [Q&A] [answered]
Hello, and a warm thank you for the great Audacious player! :)

I'm on Windows 11 and I'm very grateful for the compiled buid / installer, but I don't understand how to install the plugins... Is the compilation step mandatory, or do you plan to release compiled Windows builds, for ignorants like me? Thanks in advance for any answer! :)

EDIT: After reflection, I realize there's a possibility that the plugins could be already included in the Win builds (sorry to not have thought about that before..)... **If** it's the case, it could be interesting to indicate that in the download page on your site, always for total ignorants like me...

Sorry for this stupid question! :)

**Comment by radioactiveman** [comment, marked as answer]
Yes, the plugins are already included in the installer / ZIP file. I have added a hint on the download page as suggested.

**Reply by Artist-on-line** [reply]
Thank you so much for your answer, and sorry for the waste of time! Have good vibes! :)

**Reply by rj-rl** [reply]
> Yes, the plugins are already included in the installer / ZIP file

didn't wanna create a separate thread so instead I'm asking here: is the lastfm scrobbler plugin supposed to be included on Windows? The release notes for 4.4 mention that it's now supported on Mac and Windows, but I'm not seeing it in the list of plugins on v4.4.2

**Reply by radioactiveman** [reply]
No, it is intentionally missing on Windows for now. See also: https://github.com/audacious-media-player/audacious/issues/1383
